Tasks/AzureFileCopyV3/package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "AzureFileCopyV2",
   "dependencies": {
-    "@types/mocha": "2.2.48",
-    "@types/node": "6.0.68",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
     "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.4",
-    "azure-pipelines-task-lib": "2.8.0",
+    "azure-pipelines-task-lib": "3.0.6-preview.0",
```

Comment:
is there a stable version of this library that we can use ? Or do we have to use this version ?

Reply:
@arjgupta Hi! This is the version prepared for Node 10 migration.